pkg/helm/helmcli.go

```diff
@@ -154,6 +155,9 @@ func GetValues(kubeInfo *client.KubeInfo, releaseName string, namespace string)
 	if err != nil {
 		return nil, err
 	}
+	if vals == nil {
```

Comment:
This is not related to the orginal issue but I found this could cause nil exception as well.